packages/connect-examples/update-webextensions.js

```diff
@@ -61,13 +61,24 @@ rootPaths.forEach(dir => {
 
     const srcPath = path.join(__dirname, '../connect-web');
 
-    ['trezor-content-script.js', 'trezor-usb-permissions.js'].forEach(p => {
+    ['trezor-content-script.js'].forEach(p => {
         fs.copyFileSync(
             path.join(srcPath, 'src', 'webextension', p),
             path.join(rootPath, buildFolder, 'vendor', p),
         );
     });
 
+    ['trezor-usb-permissions.js'].forEach(p => {
+        const content = fs.readFileSync(path.join(srcPath, 'src', 'webextension', p), 'utf-8');
+        if (trezorConnectSrc !== DEFAULT_SRC) {
+            const replaced = content.replace(
+                /^const url = .*$/gm,
+                `const url = '${trezorConnectSrc}';`,
+            );
+        }
+        fs.writeFileSync(path.join(rootPath, buildFolder, 'vendor', p), replaced, 'utf-8');
```

Comment:
The variable `replaced` is defined in if scope so it does not exists out of it. And also it is missing the case when using `DEFAULT_SRC`.

```suggestion
         const content = fs.readFileSync(path.join(srcPath, 'src', 'webextension', p), 'utf-8');
        let replaced = content;
        if (trezorConnectSrc !== DEFAULT_SRC) {
            replaced = content.replace(/^const url = .*$/gm, `const url = '${trezorConnectSrc}';`);
        }
        fs.writeFileSync(path.join(rootPath, buildFolder, 'vendor', p), replaced, 'utf-8');

```